components/UserList.js

```diff
@@ -55,110 +37,102 @@ class UserList extends Component {
     if (!window.confirm(message)) {
       return
     }
-    await secureFetch(
-      `${this._getUrl()}/${user.id}`,
+    // TODO: Can we replace with useSWR (might only be possible for fetching/GET)?
+    const result = await secureFetch(
+      `${_getUrl(type)}/${user.id}`,
       accessToken,
       'delete'
     )
-    await this.fetchUserData(true)
-  }
-
-  onViewUser = (user) => {
-    const {router, type} = this.props
-    if (!user) router.push(`/manage?type=${type}`)
-    else router.push(`/manage?type=${type}&userId=${user.id}`)
+    mutate(_getUrl(type))
+    if (result.code >= 400) {
+      window.alert(result.message)
+    }
   }
-
-  onCreateUser = async () => {
-    const {auth, type} = this.props
-    const { accessToken } = auth
-    const email = window.prompt(`Enter an email address for ${type} user.`)
+  const onCreateAdminUser = async () => {
+    const email = window.prompt(`Enter an email address for admin user.`)
     // TODO: Validate user.
     if (!email) return
     // Create user and re-fetch users.
+    const adminUrl = _getUrl('admin')
+    // TODO: Can we replace with useSWR (might only be possible for fetching/GET)?
```

Comment:
Isn't mutate the way SWR does that?

Reply:
No, this TODO comment is from earlier in my research into SWR. SWR is basically only for fetching. `mutate` is used to indicate that the data fetched needs to be re-fetched, not actually to mutate the data. I was confused by this naming also.